### PR TITLE
easyengine 4.4.1

### PR DESCRIPTION
**Pull request by mrrobot47**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
Looks like it's trying to write to `/opt/easyengine` on Linux, which is why this is failing there. Is there a way we can convince this to only write to `testpath` during the test?

**Comment by mrrobot47** [comment]
@carlocab this is not meant to be run on Linux without extra privileges. Can we skip/ignore the ubuntu pipeline?

**Comment by carlocab** [comment]
Are there no commands you can run without privileges?

**Comment by mrrobot47** [comment]
>Are there no commands you can run without privileges?

On Linux, it requires the directory `/opt/easyengine`. So, it will fail in the ubuntu pipeline.

In contrast to that, on macOS, it works with `~/easyengine` directory, so it does not need any extra permissions.

**Comment by mrrobot47** [comment]
As the formula does not have any Linux bottle, can we not simply skip/ignore the ubuntu pipeline results?

**Comment by carlocab** [comment]
So nothing works at all without `sudo`? Odd. You can do this:
```diff
diff --git a/Formula/easyengine.rb b/Formula/easyengine.rb
index d61c966271b..02ee3a8d38a 100644
--- a/Formula/easyengine.rb
+++ b/Formula/easyengine.rb
@@ -24,10 +24,15 @@ class Easyengine < Formula
 
   def install
     bin.install "easyengine.phar" => "ee"
+
+    bash_completion.install "utils/ee-completion.bash" => "ee"
+    zsh_completion.install "utils/ee-completion.zsh" => "_ee"
   end
 
   test do
-    system bin/"ee config set locale hi_IN"
+    return if OS.linux? # requires `sudo`
+
+    system bin/"ee", "config", "set", "locale", "hi_IN"
     output = shell_output("#{bin}/ee config get locale")
     assert_match "hi_IN", output
 
```

**Comment by mrrobot47** [comment]
Thanks @carlocab :raised_hands: 

I have updated the tests according to your suggested patch. Not adding the completion just yet as they need some updates. Will be adding them later.

File: Formula/easyengine.rb [hunk 1]
```diff
@@ -27,6 +27,8 @@ def install
   end
 
   test do
+    return if OS.linux? # requires `sudo`
+
     system bin/"ee config set locale hi_IN"
```
Comment (carlocab) [inline Formula/easyengine.rb:32]: Let's clean this line up too, please.

Comment (carlocab) [reply Formula/easyengine.rb:32]: See patch above.

Comment (mrrobot47) [reply Formula/easyengine.rb:32]: Sorry, I had missed that. Updating.

Comment (mrrobot47) [reply Formula/easyengine.rb]: @carlocab Done with the changes. Anything else?

**Review by carlocab — APPROVED** [review]
All done. Thanks @mrrobot47!

**Pull request closed without merging**